pkg/playground/plot-pf4.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import { ChartArea, ChartContainer, ChartGroup, ChartLabel, ChartVoronoiContainer } from '@patternfly/react-charts';
+
+const Sparkline = () => {
+    return (
+        <>
+            <h3>Vertical</h3>
```

Comment:
That's an unexpected naming.. Intuitively I had called that one horizontal. But it does coincide with the attribute. :man_shrugging: